dashboards/ansible-automation-controller/dashboard.json

```diff
@@ -130,10 +130,26 @@
 						}
 					}
 				},
+				{
+					"title": "",
+					"layout": {
+						"column": 1,
+						"row": 2,
+						"width": 2,
+						"height": 4
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.markdown"
+					},
+					"rawConfiguration": {
+						"text": "**About**\n\nInstrument your Red Hat Ansible Automation Controller with New Relic - [Add data](https://one.newrelic.com/catalog-pack-details?state=4dc166c8-8662-e494-0156-c10773c90ea6)\n\nUnable to find data in your dashboard? - [Troubleshoot here](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic/)\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=AnsibleAutomationController) here and let us know how we can improve it for you.\n\nFollow New Relic [Ansible Automation Controller documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/ansible-controller-integration/) to instrument RedHat Ansible Automation Platform."
```

Comment:
```suggestion
						"text": "**About**\n\nInstrument your Red Hat Ansible Automation Controller with New Relic - [Add data](https://one.newrelic.com/catalog-pack-details?state=4dc166c8-8662-e494-0156-c10773c90ea6)\n\nUnable to find data in your dashboard? - [Troubleshoot here](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic/)\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=AnsibleAutomationController) here and let us know how we can improve it for you.\n\nFollow New Relic [Ansible Automation Controller documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/ansible-controller-integration/) to instrument Red Hat Ansible Automation Platform."
```

Reply:
Hi @mdumpati ,
As per your comment, I have changed about section content.

Thank you...